ui/tests/acceptance/secrets/backend/azure/azure-configuration-test.js

```diff
@@ -125,18 +126,15 @@ module('Acceptance | Azure | configuration', function (hooks) {
 
         await click(SES.configTab);
         await click(SES.configure);
-        await fillInAzureConfig(this.type);
+        await fillInAzureConfig();
         await click(GENERAL.saveButton);
         assert.true(
           this.flashSuccessSpy.calledWith(`Successfully saved ${path}'s configuration.`),
           'Success flash message is rendered showing the azure model configuration was saved.'
         );
         assert
           .dom(GENERAL.infoRowValue('Root password TTL'))
-          .hasText(
-            '1 hour 26 minutes 40 seconds',
```

Comment:
I messed around some with the secret-engine-helper.js method `fillInAzureConfig` so this value changed.